vnext/Microsoft.ReactNative/IReactContext.idl

```diff
@@ -11,70 +11,159 @@ import "IReactPropertyBag.idl";
+      "at the time when the React instance is created.\n"
+      "Controls whether the instance triggers the hot module reload logic when it first loads the instance.\n"
+      "Most edits should be visible within a second or two without the instance having to reload.\n"
+      "Non-compatible changes still cause full reloads.\n"
+      "See [Fast Refresh](https://reactnative.dev/docs/fast-refresh) for more information on Fast Refresh.")
     Boolean UseFastRefresh { get; };
+
+    DOC_STRING(
+      "A read-only snapshot of @ReactInstanceSettings.UseDirectDebugger property value "
+      "at the time when the React instance is created.\n"
+      "Enables debugging in the JavaScript engine (if supported).\n"
+      "For Chakra this enables debugging of the JS runtime directly within the app using "
+      "Visual Studio -> Attach to process (Script)")
     Boolean UseDirectDebugger { get; };
+
+    DOC_STRING(
+      "A read-only snapshot of @ReactInstanceSettings.DebuggerBreakOnNextLine property value "
+      "at the time when the React instance is created.\n"
+      "For direct debugging, controls whether to break on the next line of JavaScript that is executed.\n"
+      "This can help debug issues hit early in the JavaScript bundle load.\n"
+      "***Note: this is not supported with the Chakra JS engine which is the currently used JavaScript engine. "
+      "As a workaround you could add the `debugger` keyword in the beginning of the bundle.***")
     Boolean DebuggerBreakOnNextLine { get; };
+
+    DOC_STRING(
+      "A read-only snapshot of @ReactInstanceSettings.DebuggerPort property value "
+      "at the time when the React instance is created.\n"
+      "When @.UseDirectDebugger is enabled, this controls the port that the JavaScript engine debugger will run on.")
     UInt16 DebuggerPort { get; };
+
+
+    DOC_STRING(
+      "A read-only snapshot of @ReactInstanceSettings.DebugBundlePath property value "
+      "at the time when the React instance is created.\n"
+      "When loading from a bundle server (such as metro), this is the path that will be requested from the server.")
     String DebugBundlePath { get; };
+
+    DOC_STRING(
+      "A read-only snapshot of @ReactInstanceSettings.BundleRootPath property value "
+      "at the time when the React instance is created.\n"
+      "Base path used for the location of the bundle.")
     String BundleRootPath { get; };
+
+    DOC_STRING(
+      "A read-only snapshot of @ReactInstanceSettings.SourceBundleHost property value "
+      "at the time when the React instance is created.\n"
+      "When using a @.UseFastRefresh, @.UseLiveReload, or @.UseWebDebugger this is the server hostname "
+      "that will be used to load the bundle from.")
     String SourceBundleHost { get; };
+
+    DOC_STRING(
+      "A read-only snapshot of @ReactInstanceSettings.SourceBundlePort property value "
+      "at the time when the React instance is created.\n"
```

Comment:
```suggestion
      "A read-only snapshot of the @ReactInstanceSettings.SourceBundlePort property value "
      "at the time when the React instance was created.\n"
```

Reply:
Ah... It would be so much easier to mention this change in one place...